diff --git a/events/janus_wsevh.c b/events/janus_wsevh.c
index a4f24492..6e3d5997 100644
--- a/events/janus_wsevh.c
+++ b/events/janus_wsevh.c
@@ -164,7 +164,7 @@ static lws_sorted_usec_list_t sul_stagger = { 0 };
 static gint64 disconnected = 0;
 static gboolean reconnect = FALSE;
 static int reconnect_delay = 0;
-#define JANUS_WSEVH_MAX_RETRY_SECS	8
+#define JANUS_WSEVH_MAX_RETRY_SECS	1
 
 typedef struct janus_wsevh_client {
 	struct lws *wsi;		/* The libwebsockets client instance */
